integration-tests/cli/price-export.it.js

```diff
@@ -0,0 +1,210 @@
+import fs from 'fs'
+import tmp from 'tmp'
+import path from 'path'
+import csv from 'csvtojson'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'child_process'
+import {
+  sampleProductType,
+  sampleCustomerGroup,
+  sampleChannel,
+  sampleCustomType,
+  createProducts,
+} from './helpers/prices.data'
+import { createData, clearData } from './helpers/utils'
+import { version } from '../../packages/price-exporter/package.json'
+
+let projectKey
+if (process.env.CI === 'true')
+  projectKey = 'price-export-integration-test'
+else
+  projectKey = process.env.npm_config_projectkey
+
+describe('Price Exporter', () => {
+  let apiConfig
+  const productType = { typeId: 'product-type' }
+  const customerGroup = { typeId: 'customer-group' }
+  const channel = { typeId: 'channel' }
+  const customType = { typeId: 'type' }
+  const bin = './integration-tests/node_modules/.bin/price-exporter'
```

Comment:
This has worked so far. It doesn't pull from npm. The module isn't even on npm and as you can see, Travis isn't complaining 😂